### ISSUE TITLE
Улучшение рулетки

### ISSUE THREAD

**Issue by WillBegin1** [SBProX/SBProX-Dev#447]
Предлагаю сделать улучшение в рулетке, думаю это то же будет очень удобная фишка)
[bandicam 2019-07-29 22-16-06-709.zip](https://github.com/SBProX/SBProX-Dev/files/3443964/bandicam.2019-07-29.22-16-06-709.zip)

**Comment by SBProX** [comment]
К сожалению в целях безопасности мы не можем скачивать файлы и открывать их у себя.
Пожалуйста залейте в другом виде.

**Comment by WillBegin1** [comment]
Там видео, которое не загружается без архива

**Comment by Abrikotin** [comment]
![bandicam 2019-07-29 22-16-06-709](https://user-images.githubusercontent.com/53471671/62134283-e2476f80-b2e8-11e9-8682-143ea4b61b0c.gif)
Об инструменте рулетка - вот видео преобразованное в приемлемый для Вас формат.

**Comment by WillBegin1** [comment]
Спасибо за помощь)

**Comment by SBProX** [comment]
Принято.

**Comment by r35p3ct** [comment]
Да, хотелось бы значения в пунктах, а то в тиках на часовом графике одно, на дневном с другим шагом цены уже другое значение одного расстояния.
Плюс не хватает процентного значения.

**Comment by boris1717** [comment]
+1 
Очень не хватает %% в рулетке
